src/PKSim.Core/Model/IndividualParameterBySpecies.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace PKSim.Core.Model
 {
-   public class ContainerParameterBySpecies
+   public class IndividualParameterBySpecies
```

Comment:
I have renamed all the classes (Repos, FlatObjects etc.), which deal with the parameters not common for all species from
_**ContainerParameter**XXX_ to _**IndividualParameter**XXX_ (because in fact only individual parameters are handled here)